eng/XUnitWithTimeout.targets

```diff
@@ -0,0 +1,131 @@
+<!--
```

Comment:
Hopefully this entire file can be removed before we merge. Added so I could add a timeout to xunit.